Common.cs

```diff
@@ -303,14 +303,20 @@ static void chk_CheckStateChanged(object sender, EventArgs e)
 
         public static string speechConversion(string input)
         {
+            if (MainV2.comPort.MAV.cs.mode != "Auto" && MainV2.comPort.MAV.cs.mode != "Guided" && MainV2.comPort.MAV.cs.mode != "RTL")
+                input = input.Replace("Heading to Waypoint {wpn}", "");

```

Comment:
the reason I haven't committed this is because input is a user defined string, and the replace is over bearing and will only hit one condition which a simple thing like case will invalidate it